src/hotspot/os_cpu/bsd_aarch64/vm_version_bsd_aarch64.cpp

```diff
@@ -188,29 +232,30 @@ unsigned long VM_Version::os_get_processor_features() {
   id_aa64pfr0 = READ_SPECIALREG(ID_AA64PFR0_EL1);
 
   if (ID_AA64ISAR0_AES(id_aa64isar0) == ID_AA64ISAR0_AES_BASE) {
-    auxv = auxv | HWCAP_AES;
+    auxv = auxv | CPU_AES;
   }
 
   if (ID_AA64ISAR0_AES(id_aa64isar0) == ID_AA64ISAR0_AES_PMULL) {
-    auxv = auxv | HWCAP_PMULL;
+    auxv = auxv | CPU_PMULL;
   }
 
   if (ID_AA64ISAR0_SHA1(id_aa64isar0) == ID_AA64ISAR0_SHA1_BASE) {
-    auxv = auxv | HWCAP_SHA1;
+    auxv = auxv | CPU_SHA1;
   }
 
   if (ID_AA64ISAR0_SHA2(id_aa64isar0) == ID_AA64ISAR0_SHA2_BASE) {
-    auxv = auxv | HWCAP_SHA2;
+    auxv = auxv | CPU_SHA2;
   }
 
   if (ID_AA64ISAR0_CRC32(id_aa64isar0) == ID_AA64ISAR0_CRC32_BASE) {
-    auxv = auxv | HWCAP_CRC32;
+    auxv = auxv | CPU_CRC32;
   }
-
+ 
   if (ID_AA64PFR0_AdvSIMD(id_aa64pfr0) == ID_AA64PFR0_AdvSIMD_IMPL || \
```

Comment:
Minor nits: please remove new spaces on blank line here and one below. Also new blank line. Thanks.

Reply:
I copied Mikael`s original file again and with my additions it still shows 1 blank line removed and 1 blank added, which means: nothing has changed with the blank line :-) 
so I haven`t changed the commit again. 
- or am I missing something you see on blank lines  ?

Reply:
It could be your editor doing something automatically. Don't worry about it - whatever you can manage is fine.